sqldb/sqlc/migrations/000005_payments.up.sql

```diff
@@ -0,0 +1,284 @@
+-- payments table contains all the main information for a payment.
+CREATE TABLE IF NOT EXISTS payments (
+    -- The id of the mpp payment.
+    id BIGINT PRIMARY KEY,
+
+    -- payment_status is the status of the payment.
+    payment_status INTEGER NOT NULL REFERENCES payment_status_types(id) ON DELETE CASCADE,
+
+    -- payment_type is the type of the payment.
+    payment_type INTEGER NOT NULL REFERENCES payment_types(id) ON DELETE CASCADE,
+
+    -- amount_msat is the amount of the payment in milli-satoshis.
+    amount_msat BIGINT NOT NULL,
+
+    -- payment_request is the payment request of the payment.
+    payment_request BLOB UNIQUE NOT NULL,
```

Comment:
I wonder if we should split this out, as in the future we'll have more than one type of payment request encoding. I think it's also possible to actually have duplicates here: someone pays a zero value invoice multiple times.